src/notifications/notificationmanager.cpp

```diff
@@ -414,7 +414,7 @@ NotificationList NotificationManager::GetNotifications(const QString &owner)
     QHash<uint, LipstickNotification *>::const_iterator it = notifications.constBegin(), end = notifications.constEnd();
     for ( ; it != end; ++it) {
         LipstickNotification *notification = it.value();
-        if (notification->owner() == owner) {
+        if (notification && notification->owner() == owner) {
```

Comment:
Kind of working around. Debugged quite extensively and didn't find the root where the invalid id gets inserted. Enable debugs and see take lines at 343 and 370.

Feel free to propose the proper fix or fix by yourself.